Shell ESLint file incorrectly extends cli-style ESLint config

I am getting this error using the shell in combination with a custom ESLint configuration.

```bash
varl@veronika:~/dev/dhis2/apps/usage-analytics$ yarn start
yarn run v1.21.1
$ d2-app-scripts start
Generating internationalization strings... 
No i18n updates found! 
Bootstrapping local appShell... 
Copying appShell to temporary directory... 
Building app usage-analytics-app... 
 
Starting development server... 
The app usage-analytics-app is now available on port 3000 
 
Error: ESLint configuration in .eslintrc.js is invalid:
        - Property "extends" is the wrong type (expected string but got `[{"extends":["/home/varl/dev/dhis2/apps/usage-analytics/node_modules/@dhis2/cli-style/config/js/eslint-react.config.js"]}]`).
        - Property "extends[0]" is the wrong type (expected string but got `{"extends":["/home/varl/dev/dhis2/apps/usage-analytics/node_modules/@dhis2/cli-style/config/js/eslint-react.config.js"]}`).
        - "extends" should match exactly one schema in oneOf. Value: [{"extends":["/home/varl/dev/dhis2/apps/usage-analytics/node_modules/@dhis2/cli-style/config/js/eslint-react.config.js"]}].

    at validateConfigSchema (/home/varl/dev/dhis2/apps/usage-analytics/node_modules/eslint/lib/shared/config-validator.js:286:15)
[...]
 error Command failed with exit code 1.
 [ERROR] Start script exited with non-zero exit code 
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

The problem is this line: https://github.com/dhis2/app-platform/blob/master/shell/.eslintrc.js#L8

```js
const extendsArray = hasCustomConfig
    ? [require(customConfigFile)]
    : ['react-app']
```

If changed to:

```js
 const extendsArray = hasCustomConfig
     ? [customConfigFile]
     : ['react-app']
```

It correctly extends the configuration. ESLint does the `require` part internally.

Reference: https://eslint.org/docs/developer-guide/shareable-configs

--- a/shell/.eslintrc.js
+++ b/shell/.eslintrc.js
@@ -4,9 +4,7 @@ const fs = require('fs'),
 const customConfigFile = path.resolve('../../.eslintrc.js')
 const hasCustomConfig = fs.existsSync(customConfigFile)
 
-const extendsArray = hasCustomConfig
-    ? [require(customConfigFile)]
-    : ['react-app']
+const extendsArray = hasCustomConfig ? [customConfigFile] : ['react-app']
 
 module.exports = {
     extends: extendsArray,

